dataland-frontend/tests/e2e/fixtures/lksg/index.ts

```diff
@@ -0,0 +1,13 @@
+import { generateFixtureDataset } from "../FixtureUtils";
+import { generateLksgData } from "./LksgDataFixtures";
+import fs from "fs";
+import { LksgData } from "../../../../build/clients/backend";
```

Comment:
See also the other comment in regards to imports

Reply:
Repaired.